doc/source/management/data_at_rest_encryption.rst

```diff
@@ -454,6 +454,21 @@ Dumping of encrypted binary logs involves decryption, and can be done using
    relevant to binary logs, encrypted relay logs can not be dumped/decrypted
    in this way.
 
+.. rubric:: Upgrading from |Percona Server| |changed-version| to any higher version
+
+The key format in the :ref:`keyring vault plugin
+<keyring_vault_plugin>` was changed for binlog encryption in
+|Percona Server| |changed-version| release. When you are upgrading from
+|Percona Server| 5.7.20-19 to any version in |Percona Server| 5.7
+or 8.0 series, the binary log encryption will work after you complete the following steps:
```

Comment:
8.0 needs rephrasing because we have 5.7-like binlog encryption only up to 8.0.13.
Thus a procedure to upgrade to current 8.0 with upstream encryption needs to be described too

Reply:
Ready for review